### DISCUSSION TITLE
Multiple Owner Support

### DISCUSSION THREAD

**Discussion by diemaxxing** [IgorKowalczyk/discord-ssh#1] [Q&A] [answered]
Hi There,

Just Wondering, Is There Any Way To Add Multiple Owner For The SSH Bot..?

**Comment by IgorKowalczyk** [comment, marked as answer]
Currently not, but I'll try adding something like roles and permissions

**Reply by IgorKowalczyk** [reply]
**This feature was added in version 3!** 🎉

View https://github.com/IgorKowalczyk/discord-ssh/pull/226
